<?xml version="1.0"?>

<launch>
    <node pkg="physics_atv_visual_mapping" type="colorize_pcl_node_torch.py" name="coloroze_pcl_node" output="screen">
        <param name="~input_lidar_topic" value="/velodyne_1/velodyne_points"/>

        <param name="~input_img_topic" value="/multisense/left/image_rect_color"/>
        <param name="~input_pose_topic" value="/integrated_to_init"/>
        <param name="~input_camera_info_topic" value="/multisense/left/camera_info"/>

        <param name="~camera_frame" value="multisense/left_camera_optical_frame"/>
        <param name="~lidar_frame" value="vehicle"/>

        <param name="~image_compressed" value="false"/>
    </node>

    <!-- <node pkg="tf" type="static_transform_publisher" name="multisense_to_base" args="0 0 -0.1274 -0.013 0.2183 -.01 vehicle multisense/head 100"/> -->

    <!-- <node pkg="tf" type="static_transform_publisher" name="multisense_to_base" args="0 .1 -0.1274 0.013 0.2183 -.01 vehicle multisense/head 100"/> -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="multisense_to_base" args="-.093 0.103674 -0.125 0.0256 0.2485394 -0.0063011 vehicle multisense/head 100"/> -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="multisense_to_base" args="0 0 -0.1274 -0.01 0.2183 0 vehicle multisense/head 100"/> -->

    <node pkg="tf" type="static_transform_publisher" name="multisense_to_base" args="-.093 0.103674 -0.127 0.0256 0.2185394 0.0063011 vehicle multisense/head 100"/>


    <node pkg="pointcloud_colorization" type="multisense_intrinsics_pub.py" name="multisense_intrinsics_pub"/>
</launch>
